src/views/CollectionIndex/Paths.tsx

```diff
@@ -0,0 +1,18 @@
+
+export const Paths =
+  [{
+    path: '/samples',
+    text: 'View Samples',
+  },{
+    path: '/available-colors',
+    text: 'Available Colors',
+  },{
+    path: '/details',
+    text: 'View Details',
+  },{
+    path: '/specification',
+    text: 'Specification',
+  },{
+    path: '/items-included',
+    text: 'Items Included',
+  }]
```

Comment:
We dont' need another file for this. Can just have it in the same file.

Reply:
Yaa sure